packages/contracts/contracts/UserRegistry.sol

```diff
@@ -16,23 +16,14 @@ contract UserRegistry {
     */
 
     // Mapping of all users
-    mapping(address => userStruct) public users;
-
-    /*
-    * Structs
-    */
-
-    struct userStruct {
-        bytes32 ipfsHash;
-        bool isSet;
-    }
+    mapping(address => bytes32) public users;
 
     /*
     * Modifiers
     */
 
     modifier isValidUserAddress() {
```

Comment:
This modifier does not appear to be used anywhere, and the logic of it seems strange: In order to be valid, the `users` mapping for the user must be empty/zero?
If this is really needed, a comment here would be appropriate. 
https://github.com/OriginProtocol/platform/blob/36335ca374165524f18c0bb29acacf2d88e562af/packages/contracts/contracts/UserRegistry.sol#L25-L28

Reply:
@wanderingstan You're right. That was leftover from an older version of this contract. Making a PR to remove it.